.gitmodules

```diff
@@ -39,3 +39,6 @@
 	path = tenzir/integration/lib/bats-assert
 	url = https://github.com/bats-core/bats-assert
 	shallow = true
+[submodule "plugins/kafka/aux/librdkafka"]
+	path = plugins/kafka/aux/librdkafka
+	url = https://github.com/UrbanCompass/librdkafka
```

Comment:
This is a bit worrying to me:

<img width="292" alt="image" src="https://github.com/user-attachments/assets/a7602e6b-a689-4ee8-8aaf-b0bbfc4aee01">

Reply:
Not in this PR any more.